{"name": "测试输入：{'grant_type': 'password', 'username': '', 'password': 'Sutpc@2020', 'imageCode': 9527}", "status": "passed", "parameters": [{"name": "inData", "value": "{'grant_type': 'password', 'username': '', 'password': 'Sutpc@2020', 'imageCode': 9527}"}, {"name": "respData", "value": "{'error': 'invalid_grant', 'error_description': '用户名密码错误!'}"}], "start": 1623034924887, "stop": 1623034925085, "uuid": "fc36ba7b-822b-49af-aa30-b47da36cbf62", "historyId": "8d734c7188d3ec9f14951416bf7f182b", "testCaseId": "f7d6b4da24785c8730063062051f6466", "fullName": "test_login.TestLogin#test_login_s3", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "epic", "value": "TP系统"}, {"name": "story", "value": "登录接口"}, {"name": "severity", "value": "critical"}, {"name": "tag", "value": "Login(order=1)"}, {"name": "tag", "value": "用户名为空_密码正确"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-5R42TFK"}, {"name": "thread", "value": "13340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}